### PR TITLE
feat: mock data

### LINKED ISSUE #145: Generate mock mentee data to test UI with
As a developer, I should be able to populate the database quickly with 250 mentees and 125 mentors to test various components (directory, admin management).

Use something like [Faker](https://fakerjs.dev/) to generate random data for mentees and mentors. This should be separate from the seed file, which will be run in production. This populate script should be added as a script to package.json, maybe db:populate.

### PR DESCRIPTION
**Pull request by phill52**
closes #145
branched off of pr #144  because of user schema change

### CODE REVIEW

File: prisma/createData.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    },
+    update: {},
+    create: {
+      roleName,
+    },
+  });
+  return role;
+};
+
+export const createUser = async (
```
Comment (ulrokx) [inline prisma/createData.ts:17]: I'm not the biggest fan of this function having 11 arguments... Do you think you could instead make it take an object with all of these arguments as keys? I saw this problem with this function when it had 5 arguments but 11 is just too many haha.

**Review by ulrokx — APPROVED** [review]

File: prisma/createData.ts (hunk 1 above)
Comment (ulrokx) [reply prisma/createData.ts:17]: You can leave it as is but if you want you can change it. It is non critical part of code and it's too bad as is but is a bit of a smell imho

Comment (phill52) [reply prisma/createData.ts:17]: I can get to fixing that on Thursday!

**Review by ShawnAviles — APPROVED** [review]
Tested Locally, lgtm 👍 
Noticed that the top industries was sometimes empty. Wondering if we should keep it like that or now

**Pull request merged**